index.js

```diff
@@ -14,85 +15,89 @@ app.set('port', (process.env.PORT || 5000));
 
 
 function receivePullRequest(request, response) {
-    console.log('Received pull request: \n' + request.body);
-    response.send();
-    if (!isPullRequest(request)) return;
-    extractedPrDetails = extractRelevantDetails(request);
-    if (isPullRequestToCheck(extractedPrDetails) &&
-        !isValidPullRequest(extractedPrDetails)) {
-        console.log('Check Failed!');
-        var responseMsg = buildResponseMessage(extractedPrDetails);
-        commentOnPullRequest(extractedPrDetails.repo, extractedPrDetails.id,
-                             responseMsg);
-        console.log('Message to user: \n' + '"' + responseMsg + '"');
-    }
+  console.log('Received pull request: \n' + request.body);
+  response.send();
+  if (!isPullRequest(request)) return;
+  extractedPrDetails = extractRelevantDetails(request);
+  if (isPullRequestToCheck(extractedPrDetails) &&
+      !isValidPullRequest(extractedPrDetails)) {
+    console.log('Check Failed!');
+    var responseMsg = buildResponseMessage(extractedPrDetails);
+    commentOnPullRequest(extractedPrDetails.repo, extractedPrDetails.id,
+                         responseMsg);
+    console.log('Message to user: \n' + '"' + responseMsg + '"');
+  }
 }
 
 function isPullRequest(receivedJson) {
-    console.log("Pull Request field: " + '{' + receivedJson.body.pull_request
-                + '}');
-    return !!receivedJson.body.pull_request;
+  console.log("Pull Request field: " + '{' + receivedJson.body.pull_request
+              + '}');
+  return !!receivedJson.body.pull_request;
 }
 
 function extractRelevantDetails(receivedJson) {
-    repo = receivedJson.body.pull_request.base.repo.full_name;
-    action = receivedJson.body.action;
-    title = receivedJson.body.pull_request.title;
-    body = receivedJson.body.pull_request.body;
-    username = receivedJson.body.pull_request.user.login;
-    id = receivedJson.body.pull_request.number;
-    console.log('Received PR ' + id + ' "' + title + '" from: ' + username +
-                '\n Description: "' + body + '"');
-    return {repo : repo, id : id, title : title, body : body,
-            username : username, action : action};
+  repo = receivedJson.body.pull_request.base.repo.full_name;
+  action = receivedJson.body.action;
+  title = receivedJson.body.pull_request.title;
+  body = receivedJson.body.pull_request.body;
+  username = receivedJson.body.pull_request.user.login;
+  id = receivedJson.body.pull_request.number;
+  console.log('Received PR ' + id + ' "' + title + '" from: ' + username +
+              '\n Description: "' + body + '"');
+  return {repo : repo, id : id, title : title, body : body,
+          username : username, action : action};
 }
 
 function isPullRequestToCheck(prDetails) {
-    return prDetails.action == 'opened' || prDetails.action == 'edited' ||
-           prDetails.action == 'reopened' || prDetails.action == 'review_requested';
+  return prDetails.action == 'opened' || prDetails.action == 'edited' ||
```

Comment:
Only indent once, don't align.

Viz:
```javascript
return prDetails.action == 'opened' || prDetails.action == 'edited' ||
  prDetails.action == 'reopened' || prDetails.action == 'review_requested';
```

If it doesn't exceed 100 characters, do put it in a single line.